root/static/scripts/common/components/EventLocations.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import DescriptiveLink from './DescriptiveLink';
+
+const EventLocations = ({event}: {event: EventT}) => (
```

Comment:
Split this out because the same code would be copied in three different places otherwise.